internal/schema/schema_test.go

```diff
@@ -68,26 +70,19 @@ func TestLoad(t *testing.T) {
 		},
 	}
 
-	fsys := os.DirFS(fsDir)
-	ctx, cancelFunc := context.WithCancel(context.Background())
-	t.Cleanup(cancelFunc)
-
-	idx, err := index.Build(ctx, fsys)
-	require.NoError(t, err)
-
-	store := disk.NewFromIndexWithConf(idx, &disk.Conf{})
-	mgr := schema.NewWithConf(ctx, store, &schema.Conf{Enforcement: schema.EnforcementReject})
-
-	for _, tc := range testCases {
-		tc := tc
-		t.Run(tc.name, func(t *testing.T) {
-			err := mgr.CheckSchema(context.Background(), tc.url)
-			if tc.wantErr {
-				require.Error(t, err)
-				return
-			}
-			require.NoError(t, err)
-		})
+	for storeName, mgr := range mgrs {
+		for _, tc := range testCases {
+			tc := tc
+			testName := fmt.Sprintf("%s-%s", tc.name, storeName)
+			t.Run(testName, func(t *testing.T) {
+				err := mgr.CheckSchema(context.Background(), tc.url)
+				if tc.wantErr {
+					require.Error(t, err)
+					return
+				}
+				require.NoError(t, err)
+			})
+		}
```

Comment:
We could nest them so that it's easier to run the tests for a single manager if we need to (e.g. `go test -tags=test -run=TestLoad/disk` to run just the disk store tests.

```suggestion
	for storeName, mgr := range mgrs {
	        t.Run(storeName, func(t *testing.T) {
		    for _, tc := range testCases {
			  tc := tc
			  t.Run(tc.name, func(t *testing.T) {
				err := mgr.CheckSchema(context.Background(), tc.url)
				if tc.wantErr {
					require.Error(t, err)
					return
				}
				require.NoError(t, err)
			  })
			})
		}
```